xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java

```diff
@@ -174,7 +171,13 @@ public static String buildSubscriptionStreams(ProductSubscription subscription)
     }
 
     private static String buildSubscriptionStrings(List<CurrencyPair> currencyPairs, String subscriptionType) {
-        return subscriptionStrings(currencyPairs).map( s -> s + "@" + subscriptionType).collect(Collectors.joining("/"));
+        if ("depth".equals(subscriptionType)) {
```

Comment:
Thanks for your contribution, @lepus!

This looks good, but I'd like to retain the existing behaviour and make this configurable instead.

This is straightforward.  XStream core provides a means to provide configuration key-value pairs when initialising the `Exchange`: you call `setExchangeSpecificParametersItem` on `ExchangeSpecification`. Let's use that to specify the binance orderbook update frequency in ms.

Perhaps call it `Binance_Orderbook_Update_Frequency_Ms`?

If there is a minimum or maximum Binance allows, probably good idea to validate for it when starting.

Reply:
Thanks for your hints @badgerwithagun,

I've implemented the changes as you described with exchange specific parameters.

Please have a look on that.